README.rst

```diff
@@ -0,0 +1,18 @@
+Tornado Prometheus Exporter
+===========================
+
+Prometheus exporter for Tornado applications. Inspired by `Flask Prometheus Exporter <https://github.com/rycus86/prometheus_flask_exporter>`_.
+
+*This is early alpha. At least, tests, proper docs are missing. Thus, it's not published to Pypi yet.*
+
+
+Usage
+-----
+
+Instead of using ``tornado.web.Application`` as base class, use ``tornado_prometheus_exporter.Application``.
+
+
+Exported Time Series
+--------------------
+
+``tornado_http_request_duration_seconds_{count,sum,bucket}`` with as many bucket series as there are buckets.
```

Comment:
💭 The plugin also exposes the standard Python Prometheus client metrics, doesn't it? We should probably note that here.

Reply:
Not strictly. Those metrics are added due to the transitive usage of the prometheus client, thus I strongly believe documenting those metrics does not belong here. However, one could add a reference to the original package

Reply:
Done